docs/tutorial/devtools-extension.md

```diff
@@ -27,30 +27,36 @@ Using the [React Developer Tools][react-devtools] as example:
      * `~/.config/google-chrome-canary/Default/Extensions/`
      * `~/.config/chromium/Default/Extensions/`
    * on macOS it is `~/Library/Application Support/Google/Chrome/Default/Extensions`.
-1. Pass the location of the extension to `BrowserWindow.addDevToolsExtension`
+1. Pass the location of the extension to `ses.loadExtension`
    API, for the React Developer Tools, it is something like:
+   
    ```javascript
+   const { app, session } = require('electron')
    const path = require('path')
-   const os = require('os')
-
-   BrowserWindow.addDevToolsExtension(
-      path.join(os.homedir(), '/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/4.3.0_0')
-   )
+   app.on('ready', async () => {
+     const installedExtension = await session.defaultSession.loadExtension(path.join(__dirname, 'react-devtools'));
+     // Note that in order to use the React DevTools extension, you'll need to
+     // download and unzip a copy of the extension.
```

Comment:
Seems like this should be a list item between "locate the extension" and "load the extension", rather than a comment.